ios/Classes/MsalAuthPlugin.swift

```diff
@@ -236,6 +238,10 @@ extension MsalAuthPlugin {
         })
     }
     
+    private func getIDToken(result: @escaping FlutterResult) {
+
```

Comment:
This method is not required more I guess.

Reply:
In my case, the application needs to log in to get an ID token, which is then sent to our own service to get the user information via the ID token, the application does not retain the third login state, and the access token is provided by the service after the service gets the correct user information.

Reply:
Yes I understand, but my point is your function `getIDToken` contains an empty body. so this function is not required in the code

Reply:
Sorry, I misunderstood and have removed unnecessary methods.